### ISSUE TITLE
Supply Ship won't place in overworld

### ISSUE THREAD

**Issue by Moobien** [ldtteam/minecolonies#6065]
### Prerequisites 

- [ ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.3
- MineColonies Version: 0.13.437-release
- Structurize Version: 0.13.96-alpha
- Related Mods and their Versions:

## Expected behavior

to be able to place a supply ship in the overworld
## Actual behavior

Get promptly told "You are not allowed to place colonies in other dimensions. (This is changeable in the config file.)" Supply Camp unaffected

## Steps to reproduce the issue

1. 
2. 
3. 

### Logs

- latest.log: https://gist.github.com/Moobien/5192909cd9c708f647f83108054b8e3c
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by superninjakiwi** [comment]
Something to note is that there is not, in fact, an option in the config file to allow colonies in other dimensions.

**Comment by AefRal** [comment]
Playing the modpack "All the Mods 6" **1.16.4. minecolonies-0.13.437-RELEASE-universal**.
Went to place a supply ship in the Overworld and got the same "**You are not allowed to place colonies in other dimensions. (This is changeable in the config file.)**"
I did find at line 49, in _minecolonies-server.toml_ file, 
found in \Minecraft\Instances\All the Mods 6 - ATM6 - 1.16.4\defaultconfigs;
"**_#Should colonies in other dimensions be allowed? Note: This is buggy if the dimension doesn't have day/night! [Default: false]
	allowotherdimcolonies = false_**"
I set it to true, but it had no effect. I still got the same message. Not allowed.
So I tried to bypass the ship and just grabbed a townhall from creative, and was told a colony with ID 1 was nearby. 
But I hadn't made one yet. I tried using the commands to delete it, but it said Colony with ID 1 not found in this dimension.

![2020-11-21_03 54 49](https://user-images.githubusercontent.com/68062495/99875661-8d4efd00-2bae-11eb-8199-d0d806268fdf.png)
